dashboard/config/routes.rb

```diff
@@ -8,10 +8,6 @@ module OPS
 Dashboard::Application.routes.draw do
   resources :survey_results, only: [:create], defaults: { format: 'json' }
 
-  def redirect_to_teacher_dashboard
-    redirect CDO.code_org_url('/teacher-dashboard')
-  end
```

Comment:
I think these changes are a byproduct of the diff range you're viewing (including a merge from staging).  If you look at the full diff these changes don't appear.